style.css

```diff
@@ -441,17 +443,213 @@ form {
   padding-top: 20px;
 }
 
-@media only screen and (min-width: 1080px) {
+.languagesUsed li:nth-child(2),
+.languagesUsed li:nth-child(3),
+.languagesUsed li:nth-child(4) {
+  margin: 0;
+  border-left: 2px #555 solid;
+  padding: 2px 15px;
+}
+
+/* MOBILE POP UP */
+
+.popupWrapper {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  background: #191765;
+}
+
+.hideModal {
+  display: none;
+}
+
+.popupObject {
+  position: fixed;
+  top: 0;
+  height: 90vh;
+  width: 90%;
+  background: rgb(75, 67, 121, 0.9);
+  border: 0.15em solid white;
+  text-align: center;
+  margin-top: 2em;
+  margin-left: 1em;
+}
+
+.titleAndButton {
+  display: flex;
+  justify-content: space-between;
+  padding: 5% 0;
+}
+
+#popupTitle {
+  font-family: "Roboto", sans-serif;
+  font-size: 20px;
+  font-weight: bold;
+  color: white;
+  line-height: 28px;
+  margin-left: 1em;
+}
+
+.popupLanguages {
+  display: flex;
+  justify-content: center;
+}
+
+.popupLanguagesDesktop {
+  display: none;
+}
+
```

Comment:
- [ ] [OPTIONAL] **BRILLIANT WORK SO FAR 💎.** Everything is working perfectly. I have a suggestion to help make your design better. It would be great if you add this styling to your buttons `cursor: pointer;`. Then someone hovers on your buttons 👆-looking hand appears 😜🍀